functions/routes/services/FirePathway.js

```diff
@@ -0,0 +1,21 @@
+const firebase = require('firebase')
+
+const config = {
+    apiKey: process.env.REACT_APP_FB_APIKEY,
```

Comment:
How many other places in the code base are using this firebase configuration

Reply:
Since this is in two places, bringing the firebase config out to one place so that it can be reused.